common.go

```diff
@@ -25,8 +25,8 @@ const (
 )
 
 const (
-	dbNB string = "OVN_Northbound"
-	dbSB string = "OVN_Southbound"
+	DbNB string = "OVN_Northbound"
+	DbSB string = "OVN_Southbound"
```

Comment:
in go usually DB is uppercase, so DBNB/DBSB

Reply:
ok done!